app/controllers/hyku/api/v1/sessions_controller.rb

```diff
@@ -46,11 +45,24 @@ def user_admin_set_permissions(user)
             end
           end
 
-          def set_jwt_cookie(options)
-            response.set_cookie(
-              :jwt,
-              default_cookie_options.merge(options)
-            )
+          def set_jwt_cookies(user)
+            set_jwt_cookie(:jwt, value: generate_token(user_id: user.id, type: user_roles(user)), expires: 1.hour.from_now)
+            set_jwt_cookie(:refresh, value: generate_token(user_id: user.id), expires: 1.week.from_now)
+          end
+
+          def remove_jwt_cookies
+            %i[jwt refresh].each do |cookie|
+              set_jwt_cookie(cookie, value: '', expires: 10_000.hours.ago)
+              cookies[cookie] = nil
+            end
+          end
+
+          def set_jwt_cookie(name, options)
+            response.set_cookie(name, default_cookie_options.merge(options))
+          end
+
+          def generate_token(payload = {})
```

Comment:
Does the expiration need to be passed in here?  It is being hard coded to 1 hour which seems like it would work for the JWT cookie but should the refresh cookie expire in 1 week instead?